benzina/src/typed_uuid.rs

```diff
@@ -95,6 +95,15 @@ macro_rules! typed_uuid {
                 pub fn get(&self) -> ::uuid::Uuid {
                     self.0
                 }
+
+                /// Allows creating a new typed Uuid for testing
+                ///
+                /// The inner type must be declared as `pub` in the macro invocation
+                /// in order to create an instance of the typed Uuid with `.new()`
```

Comment:
This is not exact IMHO, you cannot call `new` from other modules if the actual type is not `pub`, but this is not directly related to the visibility of the function.